Tour/subscripts.md

```diff
@@ -169,20 +169,20 @@ A member subscript that accepts no argument can be declared as a _computed prope
 
 ```
 type Angle {
-  public var radians: Double
+  public var radians: Float64
   public memberwise init
   
-  public property degrees: Double {
+  public property degrees: Float64 {
     let {
-      radians * 180.0 / Double.pi
+      radians * 180.0 / Float64.pi
```

Comment:
This one looks like it could/should work, but has a type issue. Also perhaps unimplemented, but just checking.
```
.build/release/hc Examples/test.hylo -o test && ./test
Examples/test.hylo:7.25-35: error: cannot pass value of type '[{}] () let -> Float64' to parameter 'let Float64'
      radians * 180.0 / Float64.pi
                        ~~~~~~~~~~
Examples/test.hylo:10.33-43: error: cannot pass value of type '[{}] () let -> Float64' to parameter 'let Float64'
      var d = radians * 180.0 / Float64.pi
                                ~~~~~~~~~~
Examples/test.hylo:12.21-31: error: cannot pass value of type '[{}] () let -> Float64' to parameter 'let Float64'
      radians = d * Float64.pi / 180.0
                    ~~~~~~~~~~
Examples/test.hylo:15.18-27: error: undefined name 'new_value' in this scope
      &radians = new_value * Float64.pi / 180.0
                 ~~~~~~~~~
```

Reply:
The reason here is that `pi` is a method that should be called. The messages say that we're attempting to call `Float64.infix/` with a function argument (i.e., an unapplied method).

Reply:
Updated the example to call pi as a method: https://github.com/hylo-lang/GitBooks/pull/13/commits/d6c634e55ba602e63b59d670abd1cc3f1f565ae7